### ISSUE TITLE
Accessibility: Tabbing to "Cancel" button then pressing "enter" creates new notebook in error

### ISSUE THREAD

**Issue by triptych** [observablehq/feedback#165]
**Describe the bug**
In the "new notebook" dialog, if I tab to the "cancel" button, and it's highlighted. Hitting "enter" should cancel the dialog. Instead it submits and creates a new notebook.

**To Reproduce**
1. go to any notebook ( signed in ) 
2. hit 'create a new notebook' button.
3. Hit tab key till "cancel" button is focused (or tab 2x times more until "X" is focused)
4. Hit enter
**Expected behavior**
Dialog should close.

**Screenshots**
Cancel  button focused. Hitting enter still creates blank notebook. 
Same when "x" is focused.
<img width="377" alt="Screen Shot 2021-08-04 at 2 45 09 PM" src="https://user-images.githubusercontent.com/8691/128259262-c31c948a-f00e-4cdb-b0df-b1baf9053f5e.png">

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version  92.0.4515.131 (Official Build) (x86_64)

**Additional context**
Hitting "spacebar" on  Close DOES close the dialog, but then hitting spacebar on any focused Notebook template DOES NOTHING

**Comment by CobusT** [comment]
Isn't the default behavior in MacOS that the Enter key executes the default button (Create notebook) and that Space bar executes the highlighted button (which you change when tabbing through the UI elements, and is indicated by the halo/shadow around the UI element).

Eg. the 'Shut Down...' modal in MacOS.

**Comment by triptych** [comment]
If that is the case then tabbing to a notebook template and hitting space bar aught to choose that template and go. It's just not consistent.

**Comment by CobusT** [comment]
Space bar is effectively the same as clicking on the highlighted item. 
Clicking on the template does not launch the new template, it simply selects it.

**Comment by triptych** [comment]
Yeah I see same behavior. I agree with your assessment. Feel free to close this as invalid.
